MapboxNavigationTests/NavigationViewControllerTests.swift

```diff
@@ -13,14 +13,16 @@ class NavigationViewControllerTests: XCTestCase {
     
     var updatedStyleNumberOfTimes = 0
     
-    lazy var dependencies: (navigationViewController: NavigationViewController, startLocation: CLLocation, poi: [CLLocation], endLocation: CLLocation) = {
+    lazy var dependencies: (navigationViewController: NavigationViewController, startLocation: CLLocation, poi: [CLLocation], endLocation: CLLocation, voice: RouteVoiceController) = {
```

Comment:
This is cool for now, but I think this lazy var pattern is outliving its usefulness. Also, doesn't it hang around in between test runs? Or does XCTest create a new instance of the test class for each test method, as JUnit does? I forget...

Nonetheless, we're going to want to unwind/refactor this (not necessarily in this PR)